lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -19,6 +20,8 @@ class TraceMiddleware
 
         def initialize(app)
           @app = app
+
+          patch_middleware(@app)
```

Comment:
Should I just make this conditional on Ruby being version 2 and above?

Reply:
Yeah I think that's totally okay. New features don't have to support 1.9, but they should make sure they don't crash 1.9, so a conditional would be fine. Something like `if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.0')`.

Side note: using `Gem::Version` is pretty important because naive text comparison of versions can sometimes cause incorrectly valuated conditionals when the patch version increases into the double digits. (e.g. `2.1.3` vs `2.1.13`)